Project2/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">  
+    <title>MovieCard</title>
+</head>
+<body style="background-color:#8a2be2"; >
+<div class="card"> 
+<figure>
```

Comment:
Since your image doesn't have a caption or need a containing element, you can eliminate this `figure` and it's closing element.

If you had an image from the film and the "play" button was a separate icon or SVG, then you'd keep the `figure` to encapsulate/wrap them all up.